src/components/PathwayCourse/index.js

```diff
@@ -132,12 +131,16 @@ function PathwayCourse() {
   const [openSnackbar, setOpenSnackbar] = useState(false);
   const [openModal, setOpenModal] = useState(false);
   const [certificate, setCertificate] = useState("");
-  let completedAll = pathwayId==8?completedPortion?.total>=80: completedPortion?.total === 100;
+  let completedAll =
+    pathwayId == 8
+      ? completedPortion?.total >= 80
+      : completedPortion?.total === 100;
   // let completedAll = true
   const [loader, setLoader] = useState(false);
-  const displayCert = pathwayId == 1 
+  const displayCert = pathwayId == 1;
```

Comment:
Similar comment to https://github.com/navgurukul/bhanwari-devi/pull/1002/files#r1229700374. There's the lint issue of using `==` and also we want to avoid explicit references to pathway ids in case they change. So, let's maybe add a line `const IDS_OF_PATHWAYS_WITH_CERTIFICATES = ['1'];` to the constants file and then we could do the following. This would also have the added benefit of allowing us to add pathways with certificates in the future by merely updating the constants file, without touching the code. Again, it'd be good to get this information from the back-end at some point.
```suggestion
  const displayCert = IDS_OF_PATHWAYS_WITH_CERTIFICATES.includes(pathwayId);
```